chalk-solve/src/infer/ucanonicalize.rs

```diff
@@ -80,7 +81,7 @@ pub trait UniverseMapExt {
     fn add(&mut self, universe: UniverseIndex);
     fn map_universe_to_canonical(&self, universe: UniverseIndex) -> Option<UniverseIndex>;
     fn map_universe_from_canonical(&self, universe: UniverseIndex) -> UniverseIndex;
-    fn map_from_canonical<T, I>(
+    fn map_from_canonical<T: Clone, I>(
```

Comment:
Heads up: new `Clone` bounds.

Reply:
Yeah, no way around this

Reply:
Well, if we take `canonical_value` by value...

Reply:
Good poing, updated, but there are still two `clone()`s needed in `merge_answer_into_strand`.